modules/ROOT/pages/install-prereqs.adoc

```diff
@@ -0,0 +1,402 @@
+4 MBps disk read/write performance.
+
+*** This disk is referred to as the `docker` device.
+
+==== Scaling Considerations
+
+Consider scaling the number of controller nodes in the following situations:
+
+* Fault tolerance is needed to mitigate the impact of controller node hardware failure. The minimum requirement of 3 controller nodes enables fault-tolerance of losing 1 controller. To improve fault-tolerance to lose 2 controllers, a total of 5 controllers should be configured.
+* Production traffic is being served with applications running on Runtime Fabric.
+
+=== Worker Nodes
+
+Runtime Fabric requires at least three worker nodes to run Mule applications and API gateways.
+
+The maximum number of worker nodes supported is 16.
+
+Provision at least one extra worker node to preserve uptime in the event a
+node becomes unavailable (fault-tolerance) or during maintenance operations that require restarts, such as OS patching.
+
+This enables the safe removal of a worker node from the cluster to apply upgrades, without impacting the availability of applications. 
+
+[NOTE]
+====
+Runtime Fabric can reserve up to 0.5 cores per worker node for internal services. This is important 
+when determining the amount of CPU cores and worker nodes needed.
+
+For example, your Runtime Fabric could have three worker nodes and two CPU cores for each worker node. If Runtime Fabric reserves 0.3 cores per worker node, a total of 0.9 cores, the number of available cores displayed in Runtime Manager is 5.1 vCPU instead of 6 vCPU.
+====
+
+Each worker node must have the following resources:
+
+** A minimum of 2 dedicated cores. 
+** A minimum of 15 GiB memory for each dedicated core.
++
+You must consider how many Mules and tokenizers you plan to run on Runtime Fabric, and what they are licensed to deploy. Plan on approximately 0.5 cores per worker node for overhead.
+** A minimum of 80 GiB dedicated disk for the operating system.
+*** A minimum of  20 GiB for `/tmp` directory.
+*** A minimum of 8 GiB for `/opt/anypoint/runtimefabric` directory.
+*** A minimum of 1 GiB for `/var/log/` directory.
+** A minimium of 250 GiB dedicated disk with at least 1000 provisioned IOPS for Docker overlay and other internal services. This translates to a minimum of 4 MBps disk read/write performance.
++ 
+Having 250 GiB ensures there is enough space to run applications, cache docker images, provide temporary storage for running applications, and provide log storage.
+
+=== Additional Resource Requirements for Production Environments
+
+When using Runtime Fabric in a production environment:
+
+* An external load balancer must be configured to load balance external requests to the internal load balancer 
+running on each controller VM, with a health check configured for TCP port 443. A TCP load balancer with a 
+server pool consisting of each of the controller nodes is sufficient.
+* An internal load balancer that runs on at least 3 replicas to maintain availability.
```

Comment:
List item does not grammatically follow list introduction and is not parallel with peers.
Please pay attention to this going forward by re-reading your list intros and items after writing them.

```suggestion
* Your internal load balancer must run on at least three replicas, to maintain availability.
```